boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -51,19 +53,67 @@ allocates Zephyr 4kB of RAM (only for resource table: 0x9CC00000 to 0x9CC00400).
 Serial Port
 -----------
 
+A53 Cores
+^^^^^^^^^
+
+This board configuration uses single serial communication channel with the MAIN domain UART
+(MAIN_UART6, i.e. debug port).
+
+M4F Core
+^^^^^^^^
+
 This board configuration uses a single serial communication channel with the
 MCU domain UART (MCU_UART0, i.e. P2.05 as RX and P2.07 as TX).
 
 SD Card
 *******
 
+A53 Cores
+=========
+
+Download BeagleBoard.org's official `BeagleBoard Imaging Utility`_ to create bootable
+SD-card with the Zephyr image. Optionally, the Zephyr SD Card images can be downloaded from
+`bb-zephyr-images`_.
+
+M4F Core
+========
+
 Download BeagleBoard.org's official `BeagleBoard Imaging Utility`_ to create bootable
 SD-card with the Linux distro image. This will boot Linux on the A53 application
 cores. These cores will then load the Zephyr binary on the M4 core using remoteproc.
 
 Flashing
 ********
 
+A53 Core
+========
+
+The testing requires the binary to be copied to the BOOT partition in SD card.
+
+To test the A53 core, we build the :zephyr:code-sample:`hello_world` sample with the following command.
+
+.. zephyr-app-commands::
+   :board: pocketbeagle_2/am6232/a53
```

Comment:
am6254

Reply:
Again, only A0 is supported right now.
Any changes for A1 will be a separate PR.